src/llama.cpp

```diff
@@ -17025,7 +17025,7 @@ static void llama_graph_compute(
         ggml_threadpool * threadpool) {
 #ifdef GGML_USE_METAL
     if (ggml_backend_is_metal(lctx.backend_metal)) {
-        ggml_backend_metal_set_n_cb(lctx.backend_metal, n_threads);
+        ggml_backend_metal_set_n_cb(lctx.backend_metal, 1);
```

Comment:
Can this function be removed now?

Reply:
Yes, will very likely remove it before the merge - still doing some tests with the number of callbacks.